mcan/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 Tagging in git follows a pattern: `mcan/<version>`.
 
 ## [Unreleased]
+- Adhere to `filter_map_bool_then` clippy lint.
```

Comment:
Can you add a a reference to this PR like the other changelog entries?

Reply:
I thought to add it when we do the release.
But you're completely right. Better to have it in from the beginning :smiley: